src/app/app.component.html

```diff
@@ -7,8 +7,7 @@
  
 </head>
 <body>
-  <h1>LOVE UUUUUUU MUHHHHHAAAA Editing</h1>
```

Comment:
Could you please answer me Why u have changed this ?

Reply:
Keep this professional please.